gdx/src/com/badlogic/gdx/utils/JsonReader.java

```diff
@@ -657,11 +659,17 @@ private static byte[] init__json_eof_actions_0 () {
 	static final int json_en_array = 23;
 	static final int json_en_main = 1;
 
-	// line 343 "JsonReader.rl"
+	// line 348 "JsonReader.rl"
 
 	private final Array<JsonValue> elements = new Array(8);
 	private final Array<JsonValue> lastChild = new Array(8);
 	private JsonValue root, current;
+	private boolean stop;
```

Comment:
It looks like stop is private and the only way to set it in a class that extends JsonReader is to call super.stop() or super.start(). Is there any use case you can think of where users might want deeper control over the starting/stopping process, or is this fine as private and handled purely by JsonReader?

Reply:
It's pretty deep in the parser. I suppose it's possible for the user to care if the parser has been stopped. The stop signal comes from the user currently and I can't see it needing to come from inside the parser, so the user should know they've called `stop()`.